lib/rules/object-curly-spacing.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * @fileoverview Disallows or enforces spaces inside of object literals.
+ * @author Ian Christian Myers
```

Comment:
So this rule was 90% copied from another rule. Should I just remove the old ones and replace with me?